README.rst

```diff
@@ -30,6 +30,7 @@ sum** of the downloaded ``bootstrap-salt.sh`` file.
 
 The SHA256 sum of the ``bootstrap-salt.sh`` file, per release, is:
 
+- 2020.05.28: ``6b3ea15c78f01060ab12fc01c0bb18480eaf36858c7ba188b200c0fb11aac173``
```

Comment:
This also needs to be added to develop branch

Reply:
I was gonna do that right after.  I can open that now though.